nncf/experimental/torch/fx/transformations.py

```diff
@@ -122,11 +122,22 @@ def bias_update_transformation(model: torch.fx.GraphModule):
         graph = model.graph
         target_node_name = node.node_name
         graph_node = get_graph_node_by_name(graph, target_node_name)
-        if len(graph_node.users) != 1:
-            raise nncf.InternalError(f"Node with bias have {len(graph_node.users)} users, 1 expected.")
-
-        bias_node = next(iter(graph_node.users))
-        constant_update_fn(model, bias_node, value, input_port_id=1)
+        add_nodes = []
+        for user in graph_node.users:
+            if _is_add(user):
+                add_nodes.append(user)
+        if len(add_nodes) != 1:
+            raise nncf.InternalError(f"Node {graph_node.name} has {len(add_nodes)} outputs with adds, 1 expected")
+
+        bias_node = add_nodes[0]
+        with graph.inserting_before(bias_node):
+            new_constant = create_getattr_from_value(model, graph, target_node_name + "_shifted_bias", value)
+
+        args = list(bias_node.args)
+        # A bias node suppose to have constant on the second input port.
```

Comment:
This is not a case of a separate bias layer like Add. Constant may lay in any Add input port in the OpenVINO backend and the same situation is applicable for the TorchFX, I think. Shouldn't we consider this situation also?